src/project.js

```diff
@@ -37,15 +38,26 @@ function analyse (modules, walker, options) {
         }
     }, []);
 
+    if (options.skipCalculation) {
+        return {
+            reports: reports
+        };
+    }
+
     return processResults({
         reports: reports,
-    });
+    }, options.noCoreSize);
 }
 
-function processResults(result) {
+function processResults(result, noCoreSize) {
+    if (noCoreSize === null) {
+        noCoreSize = false;
```

Comment:
Can this condition be removed? You only test whether `noCoreSize` is falsy below so it is redundant, right?